decidim-proposals/spec/lib/decidim/proposals/feature_spec.rb

```diff
@@ -47,15 +47,16 @@
     let!(:proposal) { create :proposal }
     let(:feature) { proposal.feature }
     let!(:hidden_proposal) { create :proposal, feature: feature }
+    let!(:draft_proposal) { create :proposal, :draft, feature: feature }
     let!(:moderation) { create :moderation, reportable: hidden_proposal, hidden_at: 1.day.ago }
 
     let(:current_stat) { stats.find { |stat| stat[1] == stats_name } }
 
     describe "proposals_count" do
       let(:stats_name) { :proposals_count }
 
-      it "only counts not hidden proposals" do
-        expect(Decidim::Proposals::Proposal.where(feature: feature).count).to eq 2
+      it "only counts published and not hidden proposals" do
+        expect(Decidim::Proposals::Proposal.where(feature: feature).count).to eq 3
```

Comment:
Shouldn't the count remain to `2` if the draft is ignored?

Reply:
`Decidim::Proposals::Proposal.where(feature: feature).**published**.count` would equal `2`

`Decidim::Proposals::Proposal.where(feature: feature).count` equals `3` because there is one `proposal` + one `hidden_proposal` + one `draft_proposal`

the `proposals_count` equals `1` because the `hidden_proposal`  and the `draft_proposal` are ignored in the stats